knowledge-base/actions/mnajdova/github-action-required-labels/action-security.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Require Labels'
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+    is-default: false
```

Comment:
@vandana41 `is-default` is not required when adding permissions via `environment-variable-name`, please remove that line.